### PR TITLE
Added Verus fly mode.

### PR DESCRIPTION
**Pull request by ghost**
Still bypasses till this day, damn...

### CODE REVIEW

**Comment by superblaubeere27** [comment]
Verus?

**Comment by ghost** [comment]
> Verus?

Yes, it is a packet based anticheat which detects most of cheats, just like any other anticheat. However the clever guy has not patched this Fly, so why not include it?

**Comment by superblaubeere27** [comment]
I have never heard of this AntiCheat. Maybe other people did.

**Comment by ghost** [comment]
Me neither to be honest. What I know lately is that it is obviously being used on some servers and is a meme.

**Comment by superblaubeere27** [comment]
Make it a script then. I don't think that anyone needs this to be in the mainline.

**Comment by ghost** [comment]
> Make it a script then. I don't think that anyone needs this to be in the mainline.

It is actually useful when it comes to bypasses however. Your choice.

**Comment by superblaubeere27** [comment]
>
> What I know lately is that it is obviously being used on some servers and is a meme.

I didn't read the sentence properly, sry.

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleFly.kt [hunk 1]
```diff
@@ -74,6 +76,22 @@ object ModuleFly : Module("Fly", Category.MOVEMENT) {
...
     }
 
+    private object Verus : Choice("Verus", modes) {
+        val packetHandler = handler<PacketEvent> { event ->
+            if (event.packet is PlayerMoveC2SPacket) {
+                event.packet.onGround = true
+            }
+        }
+        val shapeHandler = handler<BlockShapeEvent> { event ->
+            if (event.state.block.`is`(Blocks.AIR) && event.pos.y < player.y) {
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleFly.kt:86]: Is this really required? onGround enabled on packets should be enough, I guess?

File: src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleFly.kt [hunk 2]
```diff
@@ -74,6 +76,22 @@ object ModuleFly : Module("Fly", Category.MOVEMENT) {
...
+                event.shape = VoxelShapes.fullCube()
+            }
+        }
+        val jumpEvent = handler<PlayerJumpEvent> { event ->
+            event.cancelEvent()
```
Comment (1zun4) [inline src/main/kotlin/net/ccbluex/liquidbounce/features/module/modules/movement/ModuleFly.kt:91]: Same for here.

**Comment by 1zun4secondary** [comment]
I guess I'll rename this to Walk or something. This is more like a "walk" mode.

**Pull request merged**